### ISSUE TITLE
The documentation about the width and height of resize is confusing

### ISSUE THREAD

**Issue by KasparEtter** [vseventer/sharp-cli#36]
`sharp resize --help`:

```
sharp resize <width> [height]

Resize image to width × height
```

Reading this, I would expect that the height is inferred (so that the aspect ratio is preserved) if I only pass one value. However, as far as I can judge from playing around with the command, this is not what is happening. If only one value is provided, it is taken for _both_ the width and the height. If this is intended, I'd suggest to change the documentation to something like:

```
sharp resize <width> <height>
sharp resize <width_and_height>

Resize image to width × height respectively to a square if only one value is provided
```

A similar change would then also make sense for `sharp --help`, of course. This is just a suggestion to make this tool, which I really like so far, more user-friendly. 🙂

**Comment by NicoHof** [comment]
#35

**Comment by KasparEtter** [comment]
Thanks for all the fixes! Just a small detail: As far as I can judge, https://github.com/vseventer/sharp-cli/commit/0f81ce2920e7ffc4163ec0a7dbcf54337004410e introduces a breaking change and thus it should have been released as 2.0.0. (If I understand the change correctly, it would have broken my code once upgraded but I already adapted my script by passing the same size twice instead of once for `--fit inside --withoutEnlargement`.)
